packages/react-components/src/molecules/AnalyticsControls.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import { dropdownChevronIcon } from '../icons';
+import DropdownButton from './DropdownButton';
+import { rem, tabletScreen } from '../pixels';
+
+const containerStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  gap: rem(15),
+  'span + svg': {
+    width: '11px',
+    marginRight: '10px',
+  },
+  button: {
+    padding: `${rem(8)} ${rem(16)}`,
+  },
+
+  [`@media (max-width: ${tabletScreen.min}px)`]: {
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    marginTop: rem(24),
+    width: '100%',
+  },
+});
+
+export type TimeRangeOption =
+  | '30d'
+  | '90d'
+  | 'current-year'
+  | 'last-year'
+  | 'all';
+
+const timeRangeOptions: Record<TimeRangeOption, string> = {
+  '30d': 'Last 30 days',
+  '90d': 'Last 90 days',
+  'current-year': 'This year (Jan-Today)',
+  'last-year': 'Last year',
+  all: 'Since Hub Launch',
+};
+
+interface AnalyticsControlsProps {
+  readonly timeRange: TimeRangeOption;
+  readonly href: string;
+}
+const AnalyticsControls: React.FC<AnalyticsControlsProps> = ({
+  timeRange,
+  href,
+}) => (
+  <span css={containerStyles}>
+    <strong>View as:</strong>
+    <DropdownButton
+      noMargin
+      buttonChildren={() => (
+        <>
+          <span css={{ marginRight: rem(10) }}>
+            {timeRangeOptions[timeRange]}
+          </span>
+          {dropdownChevronIcon}
+        </>
+      )}
+    >
+      {Object.keys(timeRangeOptions).map((key) => ({
+        item: <>{timeRangeOptions[key as TimeRangeOption]}</>,
+        href: `${href}?range=${key}`,
```

Comment:
I've noticed that if I'm at a specific page then I apply a filter, the pagination gets lost. I think maybe we should keep the pagination, but I'm not sure this is a requirement, maybe we could ask João or Tiff about that?

Reply:
At the moment it would make sense yeah, but as soon as we add custom ordering selection it's gonna break down. What does page 5 means if you order by amount of outputs? 

I can see how useful it could be in the current state since it allows to get to the user we want to see, but as soon as filtering by name is added it won't be useful anymore.